### DISCUSSION TITLE
Extensão do vscode

### DISCUSSION THREAD

**Discussion by Titoomf** [Flutterando/forum#35] [Dicas]
Tem uma extensão do vscode que seria o CodeSnap, que permite tira um foto do seu código ficando  mais legível e podendo compartilhar com outras pessoas em caso de tirar duvidas especificas

**Comment by 2024-jcmljunior** [comment]
Eu não conheço extensões que faça esse procedimento, no entanto existem soluções web para isso. Talvez seja útil para você.

Eu utilizo esse.
https://codebeautify.org/code-to-image-converter

**Reply by Zeca-dev** [reply]
Eu utilizo o próprio sistema operacional pra isso. Todos têm formas de tirar um print de um ponto específico da tela. Só salvar e enviar , ou enviar direto da área de transferência.

**Reply by 2024-jcmljunior** [reply]
Realmente, para poucas linhas de código, não há problema. Eu também faço desta forma. No entanto, quando falamos de código que ultrapassam o tamanho do monitor por exemplo, não é mais possível resolver com uma captura de tela. É quando eu uso o code to image converter.

Mas é isso, cada um usa da forma que o melhor atende.
O importante mesmo é resolver o problema.

**Reply by mateusfccp** [reply]
Não é melhor simplesmente copiar o código e colar como texto? Ajuda bastante a quem está te ajudando a reproduzir na máquina dele.

**Reply by 2024-jcmljunior** [reply]
Dependendo da finalidade, sim. eu por exemplo prefiro usar capturas de tela por causa dos celulares. Não é todo aplicativo que consegue formatar o código. a captura de tela facilita nesse sentido.
É como eu disse anteriormente, não tem relevância a forma que é feito desde que resolva o problema.

Existem extensões para fazer o que o @Titoomf deseja, no entanto o recurso se restringe a linguagem de programação python e se aplica a blocos de código específicos.

Apenas uma sugestão, eu recomendo o fechamento dessa questão. nada do que vier dela será a resposta para o que você esta a procura. @Titoomf

**Reply by mateusfccp** [reply]
Isso não é uma questão, é uma dica... Dá uma olhada na categoria.

**Reply by 2024-jcmljunior** [reply]
Nossa cara, eu estava me referindo a questão do autor. não a categoria em que o mesmo foi publicado. de qualquer forma, encontra-se em uma área incorreta no meu ver. A área de dicas é um lugar onde você propõem algo. como por exemplo: Exibir as suas configurações do visual studio code. Não que isso venha ao caso. Pelo que eu vi, o forum é novo. O pessoal aos poucos aprendem.

**Comment by Titoomf** [comment]
Entendo, Mas como eu especifique no titulo, isso é uma dica , uma sugestão , uma forma que eu vi para melhorar o que eu quero compartilhar em questão de organização para outra pessoa, funcionar comigo ,agora vai de você , se funciona ou não ;)

**Comment by bernardoveras** [comment]
Esta ferramenta é top. Utilizo muito quando tenho que enviar print da função completa mas ela ultrapassa o monitor!

**Comment by DEVigodE** [comment]
Link da extenção extenção CodeSnap -> [https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap](url)

![material_operator-mono](https://user-images.githubusercontent.com/13280820/194429199-e580ad7a-47c7-40f0-a450-7c78242694c9.png)

**Reply by 2024-jcmljunior** [reply]
E não é que funciona mesmo... Valeu haha
![image](https://user-images.githubusercontent.com/82642883/194772113-d6b1e99e-312a-4022-af2f-2e91fa540f2b.png)
